toolkit/scripts/utils.mk

```diff
@@ -100,7 +100,8 @@ $(foreach var,$(watch_vars),$(eval $(call depend_on_var,$(var))))
 #       to always run the "setfacl" command but not trigger a re-run of the targets
 #       depending on this target.
 $(no_repo_acl): setfacl_always_run_phony
-	@setfacl -bnR $(PROJECT_ROOT) &>/dev/null && \
```

Comment:
did you remove `&>/dev/null` on purpose? I assume this was to reduce logging noise

Reply:
In normal use it doesn't print anything (beyond the 5 lines of makefile), and I'd like to see it so I can validate its working as expected.

Reply:
Sounds good, removing that part seems like a good idea, then